src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -214,17 +214,21 @@ public InternalRequestsController(IFusionProfileResolver profileResolver)
 
                     if (!HasChanged(request.AdditionalNote, item.AdditionalNote))
                     {
-                        var requiredDepartment = item.AssignedDepartment
-                            ?? item.OrgPosition?.BasePosition?.Department;
-
-                        if (requiredDepartment is not null)
+                        if (item.AssignedDepartment is not null)
                         {
                             or.BeResourceOwner(
-                                new DepartmentPath(requiredDepartment).GoToLevel(2),
+                                new DepartmentPath(item.AssignedDepartment).GoToLevel(2),
                                 includeParents: false,
                                 includeDescendants: true
                             );
                         }
+                        else
+                        {
```

Comment:
Maybe the fallback here should use the department on the base position, so ex. TDI resource owners can't assign PDP requests.